### PR TITLE
#1152 Display start-evaluation labels

### PR DESCRIPTION
**Pull request by ermin-muratovic**

### CODE REVIEW

**Comment by christian-kreuzberger-dtx** [comment]
are those labels displayed for every event type or just for start evaluation?

File: bridge/client/app/_interceptors/http-mock-interceptor.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+const evaluationLabelsMockData = {
+  "testid": "12345",
+  "buildnr": "build17",
+  "runby": "JohnDoe"
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HttpMockInterceptor implements HttpInterceptor {
```
Comment (christian-kreuzberger-dtx) [inline bridge/client/app/_interceptors/http-mock-interceptor.ts]: Should this really be in this PR?

**Comment by ermin-muratovic** [comment]
> are those labels displayed for every event type or just for start evaluation?

The labels are displayed for every event that has a `labels` property in the payload as a key-value pair.

File: bridge/client/app/_interceptors/http-mock-interceptor.ts (hunk 1 above)
Comment (ermin-muratovic) [reply bridge/client/app/_interceptors/http-mock-interceptor.ts]: Of course not. I removed the MockInterceptor.

**Pull request merged**